src/main/java/io/cdap/plugin/SCPRemotetoRemoteAction.java

```diff
@@ -0,0 +1,293 @@
+        String verboseFlag = config.getVerboseFlag();
+
+        //Host A -> Host B
+        channel.setCommand("scp "+compressFlag+ " "+verboseFlag+
+            " " +dirFlag +" " +pathSource +" " +pathDest);
+
+        StringBuilder outputBuffer = new StringBuilder();
+        StringBuilder errorBuffer = new StringBuilder();
+
+        InputStream in = channel.getInputStream();
+        InputStream err = channel.getExtInputStream();
+
+        channel.connect();
+
+        byte[] tmp = new byte[1024];
+        while (true) {
+            while (in.available() > 0) {
+                int i = in.read(tmp, 0, 1024);
+                if (i < 0) break;
+                outputBuffer.append(new String(tmp, 0, i));
+            }
+            while (err.available() > 0) {
+                int i = err.read(tmp, 0, 1024);
+                if (i < 0) break;
+                errorBuffer.append(new String(tmp, 0, i));
+            }
+            if (channel.isClosed()) {
+                if ((in.available() > 0) || (err.available() > 0)) continue;
+                System.out.println("exit-status: " + channel.getExitStatus());
+                break;
+            }
+            try {
+                Thread.sleep(1000);
+            } catch (Exception ignored) {
+            }
+        }
+
+        LOG.debug("output: " + outputBuffer.toString());
+        LOG.info("info: " + errorBuffer.toString());
+
+        channel.disconnect();
+        session.disconnect();
+
+    }
+
+    /**
+     * The config class for {@link SCPRemotetoRemoteAction} that contains all properties that need to be filled in by
+     * the user when building a Pipeline.
+     */
+    public static class SCPRemotetoRemoteActionConfig extends PluginConfig {
+
+        @Description("Hostname or IP Address of the SSH server.")
+        @Macro
+        public String hostBastion;
+
+        @Description("Port on which SSH server is running. Defaults to 22.")
+        @Nullable
+        @Macro
+        public Integer port;
+
+        @Description("Name of the User used to login to the Bastion SSH server.")
+        @Macro
+        public String userNameBastion;
+
+        @Description("Private Key to be used to login to the Bastion SSH Server. SSH key must be of RSA type")
+        @Macro
+        public String privateKey;
+
+        @Description("Passphrase to be used with private key if passphrase was enabled when key was created. ")
+        @Macro
+        @Nullable
+        public String passphrase;
+
+        @Description("Name of the user used to login to SSH server.")
+        @Macro
+        public String userNameA;
+
+        @Description("Hostname or IP Address of the SSH server that contains the files to copy.")
+        @Macro
+        public String hostA;
+
+        @Description("Absolute path on Host A")
+        @Macro
+        public String sourcePath;
+
+        @Description("Name of the user used to login to SSH server.")
```

Comment:
This description is the same as `userNameA`. It should distinguish the differences.